### PR TITLE
refactor: fee calcs from math crate

### PR DESCRIPTION
**Pull request by Roznovjak**
Fixes: #261

### CODE REVIEW

**Review by enthusiastmartin — COMMENTED** [review]
Also exchange and direct trade needs this change.

And the idea is also to completely remove the `WithFee` trait, it is not needed.

File: pallets/lbp/src/benchmarking.rs [hunk 1]
```diff
@@ -36,6 +36,8 @@ const ASSET_B_AMOUNT: Balance = 2_000_000_000;
 const INITIAL_WEIGHT: LBPWeight = 20_000_000;
 const FINAL_WEIGHT: LBPWeight = 90_000_000;
 
+const DEFAULT_FEE: Fee = Fee {numerator: 2, denominator: 1_000};
```
Comment (enthusiastmartin) [inline pallets/lbp/src/benchmarking.rs:39]: LBP pallet does not have configurable fee ?

**Review by enthusiastmartin — CHANGES_REQUESTED** [review]
Let's merge first the #296  and reflect the changes in this PR too.

File: pallets/lbp/src/benchmarking.rs (hunk 1 above)
Comment (Roznovjak) [reply pallets/lbp/src/benchmarking.rs]: Yes, fees are configurable, but not in the config. The fee is set during pool creation.

**Review by enthusiastmartin — COMMENTED** [review]
let's resolve the build and conflcts (if any with master) and we can merge.

**Review by lumir-mrkva — APPROVED** [review]

**Pull request merged**